pkg/util/logging/error_location_hook.go

```diff
@@ -14,7 +30,7 @@ const (
 
 // ErrorLocationHook is a logrus hook that attaches error location information
 // to log entries if an error is being logged and it has stack-trace information
-// (i.e. if it originates from or is wrapped by github.com/pkg/errors)
+// (i.e. if it originates from or is wrapped by github.com/pkg/errors).
```

Comment:
Sorry for the out of context question, but wanted to validate an assumption I am making reading the code.  If a user uses `logger.Error("some error message")` but not the `logger.WithError(error)` then the `if errObj, exists = entry.Data[logrus.ErrorKey]; !exists` will return true and short circuit, correct?

Reply:
Nope, that case should be fine -- logger.Error(...) will set the `msg` and `level` fields, but not the `error` field so that check will return false. the `error` field is set either by calling `.WithError` or `.WithField("error", err)`